docs/ops/pooling/MaxPool_1.md

```diff
@@ -52,15 +52,16 @@
     * *floor*
   * **Type**: string
   * **Default value**: *floor*
+  * **Required**: *no*
```

Comment:
@lazarevevgeny am I right that we have a ticket for the MO to make required all operation attributes?
Should we have the **Required** section in the spec?

Reply:
That fact that MO will always generate all attributes does not mean that we should specify it here as required.